server/db/db.js

```diff
@@ -0,0 +1,65 @@
+const { Pool } = require('pg');
+const { HOST, USER, PASSWORD } = require('../../.env.js');
+
+const pool = new Pool({
+  host: HOST,
+  user: USER,
+  password: PASSWORD,
+  max: 20,
+  idleTimeoutMillis: 30000,
+  connectionTimeoutMillis: 2000,
+});
+
+pool.on('acquire', () => console.error('Client acquired'));
+pool.on('remove', () => console.error('Client removed'));
+pool.on('error', (err) => {
+  console.error('Unexpected error on idle client', err)
+  process.exit(-1)
+});
+
+const addRelatedArtist = function(artistId, relatedArtistId, callback) {
+  const values = [artistId, relatedArtistId];
+  const query = 'INSERT INTO related_artists (artist_id, related_artist_id) VALUES ($1, $2)';
+  pool.query(query, values, (err, data) => {
+    if (err) return callback(err, null);
+    callback(null, data);
+  });
+};
+
+const setRelatedArtists = function (artistId, relatedArtists, callback) {
+  const id = [artistId];
+  const values = [[], []];
+  relatedArtists.forEach(relatedArtist => {
+    values[0].push(artistId);
+    values[1].push(relatedArtist.id);
+  });
+  const deleteQ = 'DELETE FROM related_artists WHERE artist_id=$1';
+  const insertQ = 'INSERT INTO related_artists(artist_id, related_artist_id) SELECT * FROM UNNEST ($1::int[], $2::int[])';
```

Comment:
This works too but I recommend you look into pg-format!! Im using that

Reply:
Sweet, thanks for the tip - I'm going to refactor to that in a future PR. Pg-format looks way cleaner